docs/source/release_notes.rst

```diff
@@ -45,6 +45,7 @@ v0.4.2 Jun 23, 2021
         * Temporarily remove update checker to fix docs warnings (:pr:`1011`)
     * Testing Changes
         * Add env setting to update checker (:pr:`978`, :pr:`994`)
+        * Test woodwork on pandas 1.3.0 release candidate and make fixes (:pr:`987`)
```

Comment:
I think I'd update this and move it to the `Changes` section to reinforce we are making updates to be compatible with pandas 1.3.0, not just updating our tests.